tomviz/DataTransformMenu.cxx

```diff
@@ -139,6 +140,9 @@ void DataTransformMenu::buildTransforms()
   new AddPythonTransformReaction(
     wienerAction, "Wiener Filter", readInPythonScript("WienerFilter"), false,
     false, false, readInJSONDescription("WienerFilter"));
+  new AddPythonTransformReaction(
```

Comment:
This lines don't conform with clang-format (see the CI fail).
When running clang-format on my machine it shows it should be
```cpp
new AddPythonTransformReaction(TVminAction, "TV_Filter",
                               readInPythonScript("TV_Filter"), false, false,
                               false, readInJSONDescription("TV_Filter"));

```